docs/src/manifolds/power.md

```diff
@@ -1,5 +1,56 @@
 # Power manifold
 
+A power manifold is based on a [`Manifold`](@ref) $\mathcal M$ to build a
+$\mathcal M^{n_1 \times n_2 \times \cdots \times n_m}$.
+For the two special cases $m=1$ and $m=2$ we obtain
+one-dimensional data, i.e. vectors of points on $\mathcal M$ and matrices of images,
+respectively.
+
+## Example
+There are two ways to store the data: in a multidimensional array or in a nested array.
+
+Let's look at both for an example. Let $\mathcal M$ be `Sphere(2)` the 2-sphere. and we want
+to look at vectors of length 4.
+
+For the default, the [`ArrayPowerRepresentation`](@ref), we store the data in da multidimensional array,
+
+```@example 1
+using Manifolds #hide
+M = PowerManifold(Sphere(2),4)
+p = cat([1.0, 0.0, 0.0],
+        [1/sqrt(2.0), 1/sqrt(2.0), 0.0],
+        [1/sqrt(2.0), 0.0, 1/sqrt(2.0)],
+        [0.0, 1.0, 0.0]
+    ,dims=2)
+```
+
+which is a valid point i.e.
+
+```@example 1
+is_manifold_point(M,p)
+```
+
+An advantage of this representation is, that it is quite efficient.
+A disadvantage might be both looking at the matrix and seeing the points.
+Another problem is, that accessing a single point is ` p[:,1]` which might be unintuitive.
+
+For the [`NestedPowerRepresentation`](@ref) we can now do
+
+```@example 2
+using Manifolds #hide
+M = PowerManifold(Sphere(2), NestedPowerRepresentation(), 4)
+p = [ [1.0, 0.0, 0.0],
+      [1/sqrt(2.0), 1/sqrt(2.0), 0.0],
+      [1/sqrt(2.0), 0.0, 1/sqrt(2.0)],
+      [0.0, 1.0, 0.0]
+    ]
+```
+
+which is again a valid point so [`is_manifold_point`](@ref)`(M,p)` here also yields true.
+A disadvantage might be that with nested arrays one looses a little bit of performance.
```

Comment:
```suggestion
A disadvantage might be that with nested arrays one loses a little bit of performance.
```
I've just checked and a simple `log` is 3x faster for the manifold we have in this example for `HybridArray` than the nested representation. The difference will be even greater when `HybridArrays` can skip more allocations.